test/commands/user/list.test.ts

```diff
@@ -77,7 +77,7 @@ describe('force:user:list', () => {
       // testUser1@test.com is aliased to testUser
       const expected = [
         {
-          defaultMarker: '(A)',
```

Comment:
It seems like there are a lot more code changes for the number of tests. How come these regressions weren't found by the unit tests originally? Can we modify the test a bit to make sure it is caught in the future?